crates/nomination/src/lib.rs

```diff
@@ -277,11 +278,7 @@ impl<T: Config> Pallet<T> {
     }
 
     fn _opt_out_of_nomination(vault_id: &T::AccountId) -> DispatchResult {
-        // TODO: force refund
-        ensure!(
-            Self::get_total_nominated_collateral(vault_id)?.is_zero(),
-            Error::<T>::HasNominatedCollateral
-        );
+        ext::staking::force_refund::<T>(vault_id)?;
```

Comment:
We should probably not allow this if it puts the vault below the secure collateral threshold